src/simulation/dynamics/spacecraft/_UnitTest/test_spacecraft.py

```diff
@@ -1102,4 +1108,5 @@ def scAccumDV():
     # SCRotation(True)
     # SCTransBOE(True)
     # SCPointBVsPointC(True)
-    scOptionalRef(True, 0.001)
\ No newline at end of file
+    scOptionalRef(True, 0.001)
+    # scAccumDV()
```

Comment:
:large_orange_diamond: Github indicates that we need a single new line at the end of the file.

Reply:
Done.